multi-data-centers-in-one-city-deployment.md

```diff
@@ -71,6 +71,10 @@ member leader_priority pdName2 4
 member leader_priority pdName3 3
 ```
 
+**Note:**
```

Comment:
@en-jin19 Please pay attention to the correct markdown format, which is already used in the Chinese version.